packages/cli/src/models/compiler/solidity/SolidityProjectCompiler.ts

```diff
@@ -66,6 +66,12 @@ class SolidityProjectCompiler {
   public async call(): Promise<void> {
     await this._loadSoliditySourcesFromDir();
     await this._loadDependencies();
+
+    if (this.contracts.length === 0) {
+      log.info('No contracts found to compile.');
```

Comment:
This line will break the `master` branch. I will push a fix into this same branch if you don't mind :smiley:, and include it in `v2.4.0-rc.1`